### PR TITLE
Sync with template

### LINKED ISSUE #14: Bump vagrant-hashistack version

### LINKED ISSUE #16: Sync with origin template
offer: https://github.com/fredrikhgrelland/terraform-nomad-minio/pull/9#issue-476234355

### PR DESCRIPTION
**Pull request by Neha-Sinha2305**
closes #14 
closes #16

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -27,36 +51,48 @@ endif
 
 test: clean up
 
-example: custom_ca
+format:
+	./format.sh
```
Comment (zhenik) [inline Makefile:55]: I think pre-commit target solve that also
```suggestion
```

**Review by zhenik — CHANGES_REQUESTED** [review]

File: Makefile [hunk 2]
```diff
@@ -27,36 +51,48 @@ endif
...
 update-box:
-	@SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} vagrant box update || (echo '\n\nIf you get an SSL error you might be behind a transparent proxy. \nMore info https://github.com/fredrikhgrelland/vagrant-hashistack/blob/master/README.md#if-you-are-behind-a-transparent-proxy\n\n' && exit 2)
+	@SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} vagrant box update || (echo '\n\nIf you get an SSL error you might be behind a transparent proxy. \nMore info https://github.com/fredrikhgrelland/vagrant-hashistack/blob/master/README.md#proxy\n\n' && exit 2)
 
-proxy-to-minio:
-	consul connect proxy -service minio-local -upstream minio:9000 -log-level debug
-docker-local-build:
-	docker build docker -t fredrikhgrelland/hive:${LAST_COMMIT_HASH}
-	docker tag fredrikhgrelland/hive:${LAST_COMMIT_HASH} fredrikhgrelland/hive:test
+pre-commit: check_for_docker_binary
```
Comment (claesgill) [inline Makefile:96]:
```suggestion
pre-commit: check_for_docker_binary check_for_terraform_binary
```
Should also be fixed in template

File: Makefile (hunk 1 above)
Comment (claesgill) [reply Makefile:55]: Yes, `make pre-commit` does the same thing

File: README_template.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+<!-- markdownlint-disable MD041 -->
+<p align="center"><a href="https://github.com/fredrikhgrelland/vagrant-hashistack-template" alt="Built on"><img src="https://img.shields.io/badge/Built%20from%20template-Vagrant--hashistack--template-blue?style=for-the-badge&logo=github"/></a><p align="center"><a href="https://github.com/fredrikhgrelland/vagrant-hashistack" alt="Built on"><img src="https://img.shields.io/badge/Powered%20by%20-Vagrant--hashistack-orange?style=for-the-badge&logo=vagrant"/></a></p></p>
```
Comment (claesgill) [inline README_template.md:2]: This file should be deleted, right?

**Review by zhenik — APPROVED** [review]

**Comment by Neha-Sinha2305** [comment]
The tests that have **consul_enabled=true** and **consul_default_policy as deny** are expected to fail. Fix for that will be done in issue #19

**Review by claesgill — APPROVED** [review]
LGTM 🚀

**Pull request merged**